docs/install/upgrade.md

```diff
@@ -43,4 +43,10 @@ So, copying `notebook` and `conf` directory should be enough.
 
    ```
    bin/zeppelin-daemon.sh start
-   ```
\ No newline at end of file
+   ```
+
+# Migration Guide
```

Comment:
Could you change `#` -> `##` ? Since `#` is used only for the docs title in many other pages.